src/go/k8s/kind.yaml

```diff
@@ -4,3 +4,4 @@ nodes:
 - role: control-plane
 - role: worker
 - role: worker
+- role: worker
```

Comment:
what's the purpose of the 3rd worker?

Reply:
Could you explain to me why the test needs or prefers to run on 3 nodes?

Reply:
We have pod affinity in the sts definition. The raft requires that there is odd number of nodes. That's why I tries to run 3 node Redpanda cluster. There is still work to do on the e2e-test front. Currently kind environment maybe isn't well suited for such test. In the near future I will check what are the alternative to test external connectivity.

Reply:
No I don't need it in current version of e2e tests